pkg/beholder/client_test.go

```diff
@@ -49,6 +49,8 @@ func TestClient(t *testing.T) {
 			"byte_key_1":           []byte("byte_val_1"),
 			"str_slice_key_1":      []string{"str_val_1", "str_val_2"},
 			"nil_key_1":            nil,
+			"beholder_domain":      "TestDomain",        // Required field
+			"beholder_entity":      "TestEntity",        // Required field
 			"beholder_data_schema": "/schemas/ids/1001", // Required field, URI
```

Comment:
@4of9,  should we remove URI validation rule for `beholder_data_schema`? 
It could be just subject name in the future or something that is not URI format

Reply:
@4of9, we need to be careful with adding required fields. Before merging this all places where `Emit` is its used needs to be patched.

Reply:
@pkcll I think the intention is to still include a URI to the schema file in `beholder_data_schema`, though we're not using it to derive the subject downstream

Reply:
> Before merging this all places where Emit is its used needs to be patched.

@pkcll by this you mean all the places / external repos where the Beholder library is used?

Reply:
>  by this you mean all the places / external repos where the Beholder library is used?

yes

Reply:
we should probably make a note to look into versioning en the future